Resources/Locale/en-US/_EE/escape-menu/ui/options-menu.ftl

```diff
@@ -0,0 +1,7 @@
+ui-options-chatstack = Automatically merge identical chat messages
+ui-options-chatstack-count = { $count ->
+   *[zero] Off
+    [1] Only last message
+    [2] Last two messages
+    [3] Last three messages
+}
```

Comment:
Could also have made this be `Automatically merge the last $count identical chat messages`

Reply:
I guess, yeah. I was trying to be fancy and learn Fluent better, but that's easier z.z I'll do the thing.